specification/app/resource-manager/Microsoft.App/preview/2023-08-01-preview/Patches.json

```diff
@@ -0,0 +1,542 @@
+          {
+            "$ref": "./Builders.json#/parameters/BuilderNameParameter"
+          },
+          {
+            "$ref": "#/parameters/PatchNameParameter"
+          },
+          {
+            "name": "patchSkipConfig",
+            "in": "body",
+            "required": true,
+            "description": "Configure patcher to skip a patch or not.",
+            "schema": {
+              "description": "Configure a patch to be skipped or not by passing a boolean.",
+              "type": "boolean"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Patch skipping detail updated successfully."
+          },
+          "202": {
+            "description": "Updating patch skipping detail.",
+            "headers": {
+              "Location": {
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Common error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Patches_Skip_Configure_0": {
+            "$ref": "./examples/Patches_Skip_Configure.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.App/builders/{builderName}/patches/{patchName}/apply": {
+      "post": {
+        "tags": [
+          "Patches"
+        ],
+        "operationId": "Patches_Apply",
+        "description": "Apply a PatchResource with patch name.",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./Builders.json#/parameters/BuilderNameParameter"
+          },
+          {
+            "$ref": "#/parameters/PatchNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Patching operation completed successfully.",
+            "schema": {
+              "$ref": "#/definitions/PatchResource"
+            }
+          },
+          "202": {
+            "description": "Patching operation has been started.",
+            "schema": {
+              "$ref": "#/definitions/PatchResource"
+            },
+            "headers": {
+              "Azure-AsyncOperation": {
+                "type": "string"
+              }
+            }
+          },
```

Comment:
if this POST operation creates a patch resource, it should return 201 and a location header, to where the resource is created